src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -132,7 +144,14 @@ export const NavBarSchedule: FC<NavBarScheduleProps> = ({ isVisible }) => {
       dayStartUtcSeconds: getUnixTime(dayStart),
       personalEvents: daysEvents.filter((event) => event.isSaved),
     };
-  }, [relatedVenueEvents, userEventIds, selectedDayIndex, getEventLocation]);
+  }, [
+    relatedVenueEvents,
+    userEventIds,
+    selectedDayIndex,
+    getEventLocation,
+    today,
+    isScheduleStartDate,
+  ]);
```

Comment:
As the code is currently defined above in this PR, `today` will change on every render, and thus break memo on this every render (which we don't want).

The solution would likely be to move the relevant aspects of it within this `useMemo` (as described elsewhere in this review)